buildingmotif/building_motif/building_motif.py

```diff
@@ -44,7 +44,7 @@ def __init__(
         :default log_level: INFO
         """
         self.db_uri = db_uri
-        self.shacl_engine = shacl_engine
+        self.shacl_engine = shacl_engine or "pyshacl"
```

Comment:
Woah, i didn't know `or` was a thing in python outside of bitwise

Reply:
Yes! It's a super helpful pattern